content/_authors/nassim.merzouk.md

```diff
@@ -0,0 +1,22 @@
+---
+fullname: Nassim Merzouk #
+role: Chargé de déploiement #
+github: nassimmzk #
+avatar: # optionnel, voir ci-dessous
+# En premier, on va regarder si tu as mis une image au format prenom.nom dans /img/authors/
+# Sinon, on utilisera le lien externe du champs avatar
+# Si tu laisse ce champs vide, alors on regardera si tu as une image sur GitHub
+# Enfin, si aucune image n'est trouvée, alors la fiole beta.gouv.fr sera utilisée sur la page communauté
+link: linkedin.com/in/merzouknassim #
```

Comment:
https://www.linkedin.com/in/merzouknassim/

Reply:
Tu dois éditer le fichier pour faire les modification
Normalement ça marche avec ce lien :
https://github.com/Nassimmzk/beta.gouv.fr/edit/patch-1/content/_authors/nassim.merzouk.md?pr=%2Fbetagouv%2Fbeta.gouv.fr%2Fpull%2F3685

Reply:
Woow trop cool ! J'allais justement mettre un commentaire pour te dire que j'étais perdu 😅